react/components/Textarea/index.js

```diff
@@ -94,10 +109,22 @@ class Textarea extends Component {
           {children}
         </textarea>
 
-        {errorMessage &&
-          <div className="c-danger f6 mt3 lh-title">{errorMessage}</div>}
-        {helpText &&
-          <div className="c-muted-1 f6 mt3 lh-title">{helpText}</div>}
+        <div className="flex justify-between">
+          <div>
+            {errorMessage && (
+              <div className="c-danger f6 mt2 lh-title">{errorMessage}</div>
+            )}
+            {helpText && (
+              <div className="c-muted-1 f6 mt2 lh-title">{helpText}</div>
+            )}
+          </div>
+          {this.props.maxLength && (
```

Comment:
Is the use of some props is above 1 in the render function, you could define a `const` and then use this  variable within the returned content.
Ex:
```
render() {
const { maxLength } = this.props

return (
    ...
    {maxLength && ...}
```